src/MeetingsSDKAdapter.js

```diff
@@ -634,6 +637,13 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     try {
       const sdkMeeting = this.fetchMeeting(ID);
 
+      // eslint-disable-next-line no-console
+      console.log('ravi joinMeeting sdkMeeting ', sdkMeeting);
```

Comment:
Please remove this console logs before merge

Reply:
removed